std-bits/bench-processor/src/main/java/org/enso/benchmarks/BenchSuiteWrapper.java

```diff
@@ -0,0 +1,52 @@
+package org.enso.benchmarks;
+
+import java.util.List;
+import java.util.Optional;
+import org.graalvm.polyglot.Value;
+
+public class BenchSuiteWrapper {
+  private final BenchSuite suite;
+  private final String moduleQualifiedName;
+
+  public BenchSuiteWrapper(BenchSuite suite, String moduleQualifiedName) {
```

Comment:
You are right, it is not a good name. I have done some refactoring in 2e57598fba196f5f6f27a9d77055e25c8f0ffd67, hopefully for the better.

Reply:
Note that hiding `BenchSuite` and other interfaces is not entirely possible, since they are used from the JMH generated code, which might be in a different project.